tests/e2e/utils/cognito_bootstrap/cognito_pre_deployment.py

```diff
@@ -124,6 +124,7 @@ def create_cognito_userpool(
     cognito_userpool.create_userpool_client(
         client_name="kubeflow",
         callback_urls=[f"https://kubeflow.{subdomain_name}/oauth2/idpresponse"],
+        logout_urls=[f"https://kubeflow.{subdomain_name}"],
```

Comment:
just realized, logout url substitution to params.env is missing in e2e scripts

please test it with the change